addOns/exim/src/main/javahelp/help/contents/exim.html

```diff
@@ -29,6 +33,34 @@ <H1>Import URLs</H1>
 <br><br>
 It also supports the <a href="automation.html">Automation Framework</a>.
 
+<H1>Export</H1>
+The add-on also adds a top level "Export" menu, providing the following functionality.
+
+<H2>Export Messages to File...</H2>
+This allows you to save requests and responses to a text file. <br/>
+Select the messages to save in the History tab - use the shift key to select multiple messages.
+
+<H2>Export Response to File...</HH2>
+This allows you to save a specific response to a file. <br/>
+Select the relevant message in the History tab - 
+note that binary responses (such as images) can be saved as well as text responses.
+
+<H2>Export All URLs to File...</H2>
+This allows you to save all of the URLs accessed to a text or HTML file. <br/>
+This can be used, amongst other things, to compare the URLs available to users with 
+different roles or permissions on the same system.
```

Comment:
Suggest using [Access Control Testing](https://www.zaproxy.org/docs/desktop/addons/access-control-testing/) too?

Reply:
I did add this, with a link. However, I guess technically the link would fail in bundled help if the other add-on isn't installed. Just ignore that?

Reply:
It will fail always, links between add-ons don't work. While I linked a mention of the add-on name is enough.